crates/sui-indexer-alt-jsonrpc/src/api/transactions/mod.rs

```diff
@@ -103,9 +103,33 @@ impl QueryTransactionsApiServer for QueryTransactions {
 
         let options = query.options.unwrap_or_default();
 
-        let tx_futures = digests
-            .iter()
-            .map(|d| response::transaction(ctx, *d, &options));
+        let tx_futures = digests.iter().map(|d| {
+            let options = options.clone();
+            async move {
+                let mut tx = response::transaction(ctx, *d, &options).await;
+
+                let config = &ctx.config().transactions;
+                let mut interval = tokio::time::interval(std::time::Duration::from_millis(
+                    config.tx_digest_retry_interval_ms,
```

Comment:
nit: the name of this parameter doesn't make as much sense now, maybe it should just be `tx_retry_interval_ms` and `tx_retry_count` below?